tests/contrib/names/test_names_api.py

```diff
@@ -101,5 +100,5 @@ def test_name_pid(app, db, example_name):
     """Test name pid creation."""
     name = example_name
 
-    assert name.pid.pid_value
-    assert name.pid.pid_type == "names"
+    assert name.pid == "0000-0001-8135-3489"
```

Comment:
Many tests might fail. This is an example to showcase it actually works as expected (creates PID value from string `identifiers`, resolves)